app/airq/commands/quality.py

```diff
@@ -73,9 +74,10 @@ def _get_message(self, zipcode: Zipcode) -> typing.List[str]:
             message.append("Try these other places near you for better air quality:")
             for recommendation in recommended_zipcodes:
                 message.append(
-                    " - {} {}: {}".format(
+                    " - {} {} {} mi: {}".format(
```

Comment:
Maybe this should be parenthesized? I’m still thinking about the formatting. Let’s bring this up in the marketing channel.

Reply:
Ok talked to lauren, let's go with "CITY ZIPCODE: AQI (Distance)" @emjgreen